charts/pega/charts/constellation/values.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Parameters for Constellation operation:
+# Docker repos and tag for image
+# dockerRepos: local/constellationui/service:v2
+# dockerRepos: meshbincam.pega.com:7000/constellationui/service
+dockerRepos: YOUR_IMAGE_HERE
```

Comment:
Can this use the same repo as the other Pega containers?  Also, remove the comments with internal URLs unless they are examples.

Reply:
We will change it to use pega public repo, Could you please tell us how to specify it?
And we will remove internal URL, it is just for reference.